packages/block-library/src/paragraph/edit.js

```diff
@@ -167,6 +167,7 @@ function ParagraphBlock( {
 								'Empty block; start writing or type forward slash to choose a block'
 						  )
 				}
+				data-empty={ content ? false : true }
```

Comment:
I don't like doing this, but I didn't see a better way to accomplish it.

Reply:
I haven't explored it myself, but you might be able to use some complex data store scan to achieve the same goal, but the approach proposed it's just simpler and more performant 😄

Reply:
Thank you for the sanity check!